Mods/vcmi/config/vcmi/english.json

```diff
@@ -90,6 +90,8 @@
 	"vcmi.systemOptions.framerateButton.help"   : "{Show FPS}\n\nToggle the visibility of the Frames Per Second counter in the corner of the game window.",
 	"vcmi.systemOptions.hapticFeedbackButton.hover"  : "Haptic feedback",
 	"vcmi.systemOptions.hapticFeedbackButton.help"   : "{Haptic feedback}\n\nToggle the haptic feedback on touch inputs.",
+	"vcmi.systemOptions.enableUiEnhancementsButton.hover"  : "Interface Enhancements",
+	"vcmi.systemOptions.enableUiEnhancementsButton.help"   : "{Interface Enhancements}\n\nToggle various quality of life interface improvements.",
```

Comment:
Not very informative. Maybe enumerate at least most important points?

Reply:
This is intentional, because it would have to be constantly maintained. Should rather be in docs I think.

Reply:
I assuming option is enabled by default? If so, you can add something like you have in PR description: "... disable this option if you want classic experience from original game"

Reply:
Possibly describe 2-3 most noticeable changes,e.g. "such as large spell book, spell icons for scrolls, etc"?

And maybe add more complete version somewhere in the docs?